whois-commons/src/test/java/net/ripe/db/whois/common/rpsl/DummifierRCTest.java

```diff
@@ -0,0 +1,151 @@
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+public class DummifierRCTest {
+
+    private DummifierRC subject;
+
+    @Before
+    public void setup() {
+        subject = new DummifierRC();
+    }
+
+    @Test
+    public void dummify_organisation() {
+        final RpslObject before = RpslObject.parse(
+            "organisation:   ORG-RIEN1-TEST\n" +
+            "org-name:       Reseaux IP Europeens Network Coordination Centre (RIPE NCC)\n" +
+            "org-type:       LIR\n" +
+            "descr:          RIPE NCC Operations\n" +
+            "address:        Singel 256\n" +
+            "address:        Amsterdam\n" +
+            "address:        Netherlands\n" +
+            "phone:          +31205552222\n" +
+            "fax-no:         +31205552266\n" +
+            "admin-c:        AC1-TEST\n" +
+            "admin-c:        AC2-TEST\n" +
+            "abuse-c:        AR1-TEST\n" +
+            "mnt-ref:        TEST-MNT\n" +
+            "mnt-by:         TEST-MNT\n" +
+            "created:        1970-01-01T00:00:00Z\n" +
+            "last-modified:  2020-09-22T09:00:00Z\n" +
+            "source:         TEST");
+
+        final RpslObject after = subject.dummify(3, before);
+
+        assertThat(after.toString(), is(
+            "organisation:   ORG-RIEN1-TEST\n" +
+            "org-name:       Dummy org-name for ORG-RIEN1-TEST\n" +
+            "org-type:       LIR\n" +
+            "descr:          ***\n" +
+            "address:        ***\n" +
+            "address:        ***\n" +
+            "address:        Netherlands\n" +
+            "phone:          +31205......\n" +
+            "fax-no:         +31205......\n" +
+            "admin-c:        AC1-TEST\n" +
+            "admin-c:        AC2-TEST\n" +
+            "abuse-c:        AR1-TEST\n" +
+            "mnt-ref:        TEST-MNT\n" +
+            "mnt-by:         TEST-MNT\n" +
+            "created:        1970-01-01T00:00:00Z\n" +
+            "last-modified:  2020-09-22T09:00:00Z\n" +
+            "source:         TEST\n"));
+    }
+
+    @Test
+    public void dummify_role() {
+        final RpslObject before = RpslObject.parse(
+            "role:           Abuse Contact\n" +
+            "descr:          RIPE NCC Operations\n" +
+            "address:        Singel 256\n" +
+            "address:        Amsterdam\n" +
+            "address:        Netherlands\n" +
+            "e-mail:         abuse@ripe.net\n" +
+            "nic-hdl:        AC1-TEST\n" +
+            "remarks:        Abuse\n" +
+            "+Contact\n" +
+            "\tRole\n" +
+            "abuse-mailbox:  abuse@ripe.net\n" +
+            "mnt-by:         TEST-MNT\n" +
+            "created:        1970-01-01T00:00:00Z\n" +
+            "last-modified:  2020-09-22T09:00:00Z\n" +
+            "source:         TEST");
+
+        final RpslObject after = subject.dummify(3, before);
+
+        assertThat(after.toString(), is(
```

Comment:
Looks like this test named wrongly...it seems to be checking the dummification of _description_ and _remarks_ and not role

Reply:
The description and remarks attributes can appear in *any* object type. I didn't want to repeat the same object with variations on description and remarks, but rather test a different object type in each test case.

Reply:
same observation applies to `dummify_aut_num` test method